src/NuGet.Core/NuGet.Commands/VerifyCommand/VerifyCommandRunner.cs

```diff
@@ -58,18 +60,37 @@ public async Task<int> ExecuteCommandAsync(VerifyArgs verifyArgs)
                         _defaultFingerprintAlgorithm)).ToList();
 
                 var verifierSettings = SignedPackageVerifierSettings.GetVerifyCommandDefaultPolicy();
+                IReadOnlyList<TrustedSignerAllowListEntry> allowList = TrustedSignersProvider.GetAllowListEntries(verifyArgs.Settings, verifyArgs.Logger);
+
                 var verificationProviders = new List<ISignatureVerificationProvider>()
                 {
-                    new IntegrityVerificationProvider(),
-                    new SignatureTrustAndValidityVerificationProvider()
+                    new IntegrityVerificationProvider()
```

Comment:
I changed lines little bit make it more comparable to similar code in PackageRestore.
https://github.com/NuGet/NuGet.Client/blob/b8fe2a0f84a2a1a7f4efe2dcaaa7876bef8f4627/src/NuGet.Core/NuGet.Packaging/PackageExtractor.cs#L998-L1011